build/config.js

```diff
@@ -1,49 +1,20 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.EnvConfig = void 0;
-// TODO: this config we may want to keep in a Core Config spreadsheet
-exports.EnvConfig = {
-    dev: {
-        coreConfig: {
-            GSPREAD_CORE_PERSON_ID: '1X5fJDmGKJtbAanHOqyQ3RCHMP63ABQK4fgpvzm9RSLM',
-            GSPREAD_CORE_ORG_ID: '1Y1RerSbgdrVcwocvCjWo8SyxutLr7BLiuJnUJlajodI',
-            GSPREAD_CORE_TYPE_ID: '1MW2utNJZcuJad2LC57D5DDTx_S_Q0iXYNg7XwlzPcx0',
-            GSPREAD_CORE_CONFIG_ID: '1rPyY8Gn7lA59GLLMAQOtCeW47zi5gA2elVcv0zMp-2M'
-        },
-        'davis.nightmarket': {
-            NM_ID: 'davis.nightmarket',
-            DISCORD_GUILD_ID: '',
-            GSPREAD_OPS_ID: '1bTt7dVKTTMY7iMNcxY0-X2Ulkw7B5p-_lmwMTDgyGP4',
-            GSPREAD_FOODCOUNT_ID: '1DPPmJU1w34PEWB3XZKeIBA0Qc7nx3R2xiJFwDbTxeNk'
-        }
-    },
-    // these point to TEST data in night-tech folder
-    test: {
-        coreConfig: {
-            GSPREAD_CORE_PERSON_ID: '1X5fJDmGKJtbAanHOqyQ3RCHMP63ABQK4fgpvzm9RSLM',
-            GSPREAD_CORE_ORG_ID: '1Y1RerSbgdrVcwocvCjWo8SyxutLr7BLiuJnUJlajodI',
-            GSPREAD_CORE_TYPE_ID: '1MW2utNJZcuJad2LC57D5DDTx_S_Q0iXYNg7XwlzPcx0',
-            GSPREAD_CORE_CONFIG_ID: '1rPyY8Gn7lA59GLLMAQOtCeW47zi5gA2elVcv0zMp-2M'
-        },
-        'davis.nightmarket': {
-            NM_ID: 'davis.nightmarket',
-            DISCORD_GUILD_ID: '',
-            GSPREAD_OPS_ID: '1bTt7dVKTTMY7iMNcxY0-X2Ulkw7B5p-_lmwMTDgyGP4',
-            GSPREAD_FOODCOUNT_ID: '1DPPmJU1w34PEWB3XZKeIBA0Qc7nx3R2xiJFwDbTxeNk'
-        }
-    },
-    prod: {
-        coreConfig: {
-            GSPREAD_CORE_PERSON_ID: '',
-            GSPREAD_CORE_ORG_ID: '',
-            GSPREAD_CORE_CONFIG_ID: '',
-            GSPREAD_CORE_TYPE_ID: ''
-        },
-        'davis.nightmarket': {
-            NM_ID: 'davis.nightmarket',
```

Comment:
if you are going to get rid of the object mapping in favor of an array, I think that's good and forward thinking, but I am not in favor of discarding NM_ID. The reason is that I do not feel confident building instance resources on guild ID, which is subject to change. I would rather guild id and all other resources be built on a human-readable identifier per instance, hence NM_ID.

Reply:
I'll add it back in

Reply:
OK, but may as well keep the array approach if that's good for you - this is how the data will come from a core config doc. In fact, we could implement that as part of this streamline effort if you like.